src/ol/image.js

```diff
@@ -122,10 +122,11 @@ ol.Image.prototype.handleImageLoad_ = function() {
 
 
 /**
- * Load not yet loaded URI.
+ * Load the image or retry if loading previously failed.
```

Comment:
Maybe add a note that loading is taken care of by the tile queue, and calling this method is only needed for preloading or for reloading in case of an error.